tests/test_token_blacklist.py

```diff
@@ -190,3 +191,13 @@ def test_jti_field_should_contain_uuid_hex_strings(self):
         actual_hexes = [i.jti_hex for i in OutstandingToken.objects.all()]
 
         self.assertEqual(actual_hexes, self.expected_hexes)
+
+
+class TokenVerifySerializerShouldHonourBlacklist(TestCase):
```

Comment:
Test case should have one that overrides the settings to try without blacklist app installed and another should try when it is.